.scrutinizer.yml

```diff
@@ -1,7 +1,7 @@
 filter:
     excluded_paths:
         - "*/Tests/*"
-        - "app/DoctrineMigrations/*"
```

Comment:
Is this not included in the previous line? Or is it case sensitive?

Reply:
Thanks, I was mislead by the removed line. I later found out that the newly added line was not having the effect I looked for. And I'm going to remove it once again. But your feedback is interesting, I do not think the excluded path regex is not performed in a case insensitive manner.